scripts/check-public-api.spec.ts

```diff
@@ -168,6 +169,24 @@ describe('check-public-api', () => {
       mock.restore();
     });
 
+    it('considers',async ()=>{
```

Comment:
[q] Isn't this test similar to the one above (which also checks referenced imports?

Reply:
ups needs to be delete